chrome/content/zotero/collectionTree.jsx

```diff
@@ -1567,6 +1568,21 @@ var CollectionTree = class CollectionTree extends LibraryTree {
 		// Directly on a row
 		if (orient === 0) {
 			if (!treeRow.editable) {
+				// Can drop collections, searches and items into trash
```

Comment:
`searches and` → `searches, and` (house style)

Reply:
We are in the oxford comma camp, got it!